cdk/visit/lambda_code/log_visit/log_visit.py

```diff
@@ -101,20 +123,32 @@ def registrationWorkflow(self, current_user):
     def addVisitEntry(self, current_user, location):
         # Get the current date at which the user logs in.
         visit_date = datetime.datetime.now().timestamp()
+        # Convert visit_date to human-readable format
+        visit_date = datetime.datetime.fromtimestamp(
+            visit_date).strftime('%Y-%m-%d %H:%M:%S')
 
         # Add the item to the table.
-        response = self.visits.put_item(
+        visits_resposne = self.visits.put_item(
             # PK = Partition Key = Visit Date
             # SK = Sort Key = Username or Email Address
 
+            Item={
+                'visit_time': str(visit_date),
+                'username': current_user,
+                'location': location,
+            },
+        )
+
+        # TODO: Should we return this?
```

Comment:
yep! Effectively we want **everything** to behave exactly as it was before. We just want to write items to the new table as well